epochStart/metachain/systemSCs.go

```diff
@@ -139,7 +138,7 @@ func (s *systemSCProcessor) processWithNewFlags(
 	}
 
 	if s.enableEpochsHandler.IsFlagEnabled(common.StakingV4Step1Flag) {
-		err := s.stakeNodesFromQueue(validatorsInfoMap, math.MaxUint32, header.GetNonce(), common.AuctionList)
+		err := s.unStakeAllNodesFromQueue()
```

Comment:
does the `stakeNodesFromQueue` is still used?

Reply:
in old code yes.